src/FiltersData.tsx

```diff
@@ -2,31 +2,31 @@ export enum FILTER_OPTIONS {
     TOP="Top Headlines",
     EVERYTHING="Everything"
 }
+export const filterOptions = ["Top Headlines", "EveryThing"];
 
 export const HeadLinesFilters = [
     {
         name: "Country",
         list: [],
-        
+        date: false
```

Comment:
instead of giving false all the time pass nothing and on the component itself make a default of "false" if nothing is passed